providers/aws/resources/aws_accessanalyzer.go

```diff
@@ -70,13 +66,14 @@ func (a *mqlAwsAccessAnalyzer) getAnalyzers(provider *aws_provider.Provider) []*
 					return nil, err
 				}
 				for _, analyzer := range analyzers.Analyzers {
-					mqlAnalyzer, err := a.MotorRuntime.CreateResource("aws.accessanalyzer.analyzer",
-						"arn", core.ToString(analyzer.Arn),
-						"name", core.ToString(analyzer.Name),
-						"status", string(analyzer.Status),
-						"type", string(analyzer.Type),
-						"tags", core.StrMapToInterface(analyzer.Tags),
-					)
+					mqlAnalyzer, err := a.MqlRuntime.CreateResource(a.MqlRuntime, "aws.accessanalyzer.analyzer",
+						map[string]*llx.RawData{
+							"arn":    llx.StringData(toString(analyzer.Arn)),
```

Comment:
No need to add a new func. The `core.ToX` moved to `convert.ToX`.

You could use `convert.ToString` or `llx.StringDataPtr` here.

Reply:
oh nice, thank you